test/WebSites/BasicWebSite/Controllers/ContentNegotiation/NormalController.cs

```diff
@@ -1,28 +1,70 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Buffers;
 using BasicWebSite.Formatters;
 using BasicWebSite.Models;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
 
 namespace BasicWebSite.Controllers.ContentNegotiation
 {
     public class NormalController : Controller
     {
+        private JsonOutputFormatter _indentingFormatter;
+
+        public NormalController(IOptions<MvcJsonOptions> jsonOptions, ArrayPool<char> charPool)
+        {
+            var defaultSettings = jsonOptions.Value.SerializerSettings;
+            var settings = new JsonSerializerSettings
```

Comment:
I wouldn't go to all the trouble of copying all the settings if the intent of the test is to prove that you can override the formatter per controller, I would just set anything that is not a default value.